src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -39,66 +39,251 @@
+      strategy_exp / exp_sum AS bandit_weight_high_temperature,
+      run_count,
+      strategy_selection_method
+    FROM (
+      SELECT
+        EXP(strategy_avg_edges / temperature) AS strategy_exp,
+        SUM(EXP(strategy_avg_edges / temperature)) OVER() AS exp_sum,
+        strategy,
+        run_count,
+        strategy_selection_method
+      FROM (
+        SELECT
+          /* Standardize the new edges data and take averages per strategy. */ AVG((new_edges - overall_avg_new_edges) / overall_stddev_new_edges) AS strategy_avg_edges,
+          strategy,
+          /* Change temperature parameter here. */ .75 AS temperature,
+          COUNT(*) AS run_count,
+          "multi_armed_bandit_high" AS strategy_selection_method
+        FROM (
+          SELECT
+            fuzzer,
+            CONCAT(s_radamsa, s_max_len, s_ml_rnn, s_vp, s_fork, s_subset, s_recommended_dict) AS strategy,
+            fuzzer_stddev,
+            AVG(new_edges) OVER() AS overall_avg_new_edges,
+            STDDEV(new_edges) OVER() AS overall_stddev_new_edges,
+            new_edges,
+            strategy_selection_method
+          FROM (
+            SELECT
+              fuzzer,
+            IF
+              (strategy_corpus_mutations_radamsa > 0,
+                "corpus_mutations_radamsa,",
+                "") AS s_radamsa,
+            IF
+              (strategy_random_max_len > 0,
+                "random_max_len,",
+                "") AS s_max_len,
+            IF
+              (strategy_corpus_mutations_ml_rnn > 0,
+                "corpus_mutations_ml_rnn,",
+                "") AS s_ml_rnn,
+            IF
+              (strategy_value_profile > 0,
+                "value_profile,",
+                "") AS s_vp,
+            IF
+              (strategy_fork > 0,
+                "fork,",
+                "") AS s_fork,
+            IF
+              (strategy_corpus_subset > 0,
+                "corpus_subset,",
+                "") AS s_subset,
+            IF
+              (strategy_recommended_dict > 0,
+                "recommended_dict,",
+                "") AS s_recommended_dict,
+              STDDEV(new_edges) OVER(PARTITION BY fuzzer) AS fuzzer_stddev,
+              new_edges,
+              strategy_selection_method
+            FROM
+              libFuzzer_stats.TestcaseRun
+            WHERE
+              ((strategy_mutator_plugin = 0)
+                OR (strategy_mutator_plugin IS NULL))
+              AND /* Query results from the past 30 days. Change as needed. */ DATE_DIFF(CAST(CURRENT_TIMESTAMP() AS DATE), CAST(_PARTITIONTIME AS DATE), DAY) < 6 )
+          WHERE
+            /* Filter for unstable targets. */ fuzzer_stddev < 150)
+        WHERE
+          strategy_selection_method = "multi_armed_bandit_high"
+        GROUP BY
+          strategy))
+    ORDER BY
+      bandit_weight_high_temperature DESC) b
+  ON
+    a.strategy = b.strategy) d
+ON
+  c.strategy = d.strategy
 """
```

Comment:
I left the query as joining the tables together, it's a lot longer but it uses the format of the original query and makes it easier to filter based on strategy_selection_method.